sherpa/astro/flux.py

```diff
@@ -160,8 +160,10 @@ def print_sample_result(title, arg):
         for x in oflxiflx:
             sf = numpy.sort(x)
             median = numpy.median(sf)
-            upconfidence = sf[(1.0 - myconfidence) * size - 1]
-            loconfidence = sf[myconfidence * size - 1]
+            upconfidence_index = int((1.0 - myconfidence) * size - 1)
```

Comment:
As noted in the code review, I've added an issue to track this change (#286), so it isn't forgotten.